style-sheets/primevue-components-overrides.scss

```diff
@@ -115,6 +115,11 @@ label {
     );
 }
 
+.#{variables.$primevue-prefix}-button:focus {
+    box-shadow: 0 0 0 2px #ffffff,
+        0 0 0 4px map.get(lightTheme.$light-theme, 'focus'), 0 1px 2px 0 black;
+}
+
```

Comment:
The styling is working fine but shouldn't the size unit be rem for consistency?

Reply:
thanks Jota, you're right, I was copying from our old code, will change that

Reply:
Hi Jota, I changed the effect a bit, please check again, thanks!!